packages/credential-governance/src/types/domain-definitions.ts

```diff
@@ -44,6 +44,7 @@ export interface IFieldDefinition {
   maxValue?: number;
   minDate?: Date;
   maxDate?: Date;
+  schema?: string;
```

Comment:
- I think a little documentation here couldn't hurt
- I think that using an object (represented by `Record<string, unknown>`) type, rather than a `string`, is better as it could allow us to conceivably change the type to the `ajv` type as @JGiter suggested without needing to migrate the role definitions.
```suggestion

  /**
   * A JSON Schema definition
   * For more information about JSON Schema, see https://json-schema.org/
   */
  schema?: Record<string, unknown>;
```

Reply:
so we do not use JSON.stringify for this case anymore? We will store real object here?